deploy/backplane/srep/hypershift/service-cluster/20-srep.SubjectPermission.yml

```
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-srep-service-cluster
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - allowFirst: true
+    clusterRoleName: backplane-srep-service-cluster-project
+    namespacesAllowedRegex: "(^uhc-.*)"
+  - allowFirst: true
+    clusterRoleName: dedicated-readers
+    namespacesAllowedRegex: "(^hypershift-*|^backplane$|^uhc-.*|ocm-*|klusterlet*|open-cluster-management-agent*|^.*-operator$)"
```

```suggestion
    namespacesAllowedRegex: "(^hypershift-.*|^backplane$|^uhc-.*|^ocm-.*|^klusterlet.*|^open-cluster-management-agent.*|^.*-operator$)"
```

Provide justification in JIRA and and in a reply here why ^.*-operator$ is necessary. This is very broad.

---

@jewzaam  In general, I considered ^`.*-operator$` would be necessary for SREP, if you think we shouldn't be allowing it, I can go ahead to remove it. I will wait for your decision call on that. Thanks!

---

Quick review of an existing service cluster and there are no NS matching `^.*-operators$` that are not already covered by `^openshift-.*`, part of https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/srep/20-srep.SubjectPermission.yml

Suggestion below includes removal of backplane regex as the namespaces are covered in the above `^openshift-.*` as well.  

Update ocm regex to include namespace without a dash, since there is the `ocm` namespace. 

Adjusted the OCM regex to remove -agent so it matches all the namespaces.  Note rbac to `redhat-open-cluster-management` is covered in by the SP linked earlier in this comment.

```suggestion
    namespacesAllowedRegex: "(^hypershift.*|^uhc-.*|^ocm.*|^klusterlet.*|^open-cluster-management.*)"
```